doc-content/enterprise-only/optimizer/quarkus-compiling-proj-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='proc-quarkus-compiling-project_{context}']
+
+
+= Running the school timetable application
+
+After you have created the school timetable project, run it in development mode. In development mode, you can update the application sources and configurations while your application is running. Your changes will appear in the running application.
+
+.Prerequisites
+* You have created the school timetable project.
```

Comment:
[style] Remove "have".

[style] Rephrase as per other prerequisites perhaps?

The school timetable project is created.

Reply:
What is your objection to have created?